main.go

```diff
@@ -87,6 +88,7 @@ func loadAMQPConfig() {
 	amqpURI = cfg.GetString("amqp.uri")
 	amqpExchangeName = cfg.GetString("amqp.exchange.name")
 	amqpExchangeType = cfg.GetString("amqp.exchange.type")
+	amqpQueuePrefix = cfg.GetString("amqp.queue_prefix")
```

Comment:
You should add something to `defaultConfig` (above) for this configuration value. Probably just "templeton" to mimic the old behavior.